src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringArray = sourceString.split(';');
+  const stylesObject = {};
+
+  const arr = stringArray.map(n => n.trim())
+    .filter(n => n.length > 0).map(m => m.split(':'));
+
+  arr.forEach(n => {
```

Comment:
Что такое n? Давай понятные имена
```suggestion
  arr.forEach([key, value] => {
```
А еще можно сразу деструктуризировать оттуда свойства